Scripts/update-translations.rb

```diff
@@ -92,11 +91,11 @@ def copy_comment(f, trans_strings, value)
 
 fallback_lang_dir = File.join('Simplenote', "#{FALLBACK_LANG}.lproj")
 
-langs.each do |code,local|
+langs.each do |code, local|
```

Comment:
💄  Nitpick: this variable should be named `locale`, not `local`.

Reply:
Fixed in https://github.com/Automattic/simplenote-macos/commit/9955f5b295aebbda78c0cf680dada45512b31fe9